src/components/Table/TableDetailWizard/TableDetailWizard.jsx

```diff
@@ -45,9 +46,9 @@ const StyledFooter = styled.div`
 `;
 
 const StyledContentContainer = styled.div`
-  padding-top: 30px;
-  padding-right: 50px;
-  padding-left: 40px;
+  padding-top: ${spacing07};
+  padding-right: ${spacing09};
```

Comment:
Same comment as above. This could cause a break in layout by adding pixels .

Reply:
This one also does not change anything significant 👍